packages/bloc/lib/src/bloc_base.dart

```diff
@@ -44,12 +44,16 @@ abstract class ErrorSink implements Closable {
   void addError(Object error, [StackTrace? stackTrace]);
 }
 
+/// The basic interface with minimal access to both:
+/// synchronous access to the current [state] and states emitting.
+abstract class BlocBaseInterface<State>
```

Comment:
I think this requires further discussion before making any code changes. I'd prefer not to add `Interface` as a suffix for example since that is inconsistent with other interfaces within the library. Closing this PR for now until we can come up with a naming convention that is generally agreed upon and consistent with the current codebase.